utilitydb.cpp

```diff
@@ -5,78 +5,105 @@ UtilityDB::UtilityDB() { establishConnection(); }
 bool UtilityDB::isConnected() { return db.isOpen(); }
```

Comment:
Do these methods make sense? There is only one return statement and that's all.

Reply:
And how will you check on the start, whether connection is established?

Reply:
I must learn how to communicate in English. Here I meant that this method we use in if statements, so can we just write if (db.isOpen())?

Reply:
I am  ~~a piece of dog shit~~, approve.